### ISSUE TITLE
Words can contain astrophope

### ISSUE THREAD

**Issue by tbm** [jwilk/anorack#7]
anorack doesn't like words that contain an astrophope, which is possible in I think the middle east.

```
Esra'a is ...
Esra’a is ...
```

```
d:2: a is -> an is /Iz/
d:3: a is -> an is /Iz/
```

**Comment by tbm** [comment]
Better example:

```
Esra’a Al Shafei
```

```
d:1: a Al -> an Al /'al/
```

**Comment by jwilk** [comment]
Fixed in [0.2.7](https://github.com/jwilk/anorack/releases/tag/0.2.7).
